examples/python/xds/README.md

```diff
@@ -3,11 +3,8 @@ gRPC Hostname Example
 
 The hostname example is a Hello World server whose response includes its
 hostname. It also supports health and reflection services. This makes it a good
-server to test infrastructure, like load balancing.
-
-The example requires grpc to already be built. You are strongly encouraged
-to check out a git release tag, since there will already be a build of gRPC
-available.
+server to test infrastructure, like load balancing .This example depends on a
+gRPC version of 1.28.1 or newer.
 
 ### Run the example
 
```

Comment:
The instructions in the "Run the example section" are a bit confusing.
There are 3 numbered steps but after that there's a big section on  configuring the server and then running a client.
It's not clear whether all the steps belong under section 3. 
Once done with the client, the docs talk about testing the server with grpcurl. Does that still belong to section 3? (probably not). I think the way the instructions are divided into sections needs some love.

Reply:
Very nice, thanks for addressing this.